Source/AwsCommonRuntimeKit/mqtt/MqttRequestResponseClient.swift

```diff
@@ -337,7 +401,7 @@ public class MqttRequestResponseClient {
   /// - Parameters:
   ///     - operationOptions: configuration options for request response operation
   /// - Returns: MqttRequestResponse
-  /// - Throws: CommonRuntimeError.crtError if submit failed
+  /// - Throws:CommonRuntimeError.crtError if submit failed
```

Comment:
Trivial: Shouldn't there be a space between Throws: and `CommonRuntimeError.crtError` ?